src/VisualStudio/Core/Def/Implementation/ProjectSystem/VisualStudioWorkspaceImpl.RenameDocumentUndoUnit.cs

```diff
@@ -0,0 +1,53 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System;
+using System.Linq;
+using Microsoft.VisualStudio.OLE.Interop;
+
+namespace Microsoft.VisualStudio.LanguageServices.Implementation.ProjectSystem
+{
+    internal partial class VisualStudioWorkspaceImpl
+    {
+        private class RenameDocumentUndoUnit : IOleUndoUnit
+        {
+            private readonly VisualStudioWorkspaceImpl _workspace;
+            private readonly string _fromName;
```

Comment:
Unused?

Reply:
Used for the description, though I'm unable to actually get that description to be used anywhere in practice.